plugins/search-backend/README.md

```diff
@@ -0,0 +1,13 @@
+# search
+
+Welcome to the search backend plugin!
+
+_This plugin was created through the Backstage CLI_
+
+## Getting started
+
+Your plugin has been added to the example app in this repository, meaning you'll be able to access it by running `yarn start` in the root directory, and then navigating to [/search](http://localhost:3000/search).
+
+You can also serve the plugin in isolation by running `yarn start` in the plugin directory.
+This method of serving the plugin provides quicker iteration speed and a faster startup and hot reloads.
+It is only meant for local development, and the setup for it can be found inside the [/dev](/dev) directory.
```

Comment:
If I'm not mistaken, the boilerplate text here from the `yarn create-plugin` call out only applies to frontend plugins?  i.e., a user would not actually add this to the app dir and run it with yarn start, etc?

I know this is a draft so being fair, but before publishing may want to tweak this. (And there may be nuance here where I'm off base, but was how I understood it!)

Reply:
Great callout!  I tried centralizing the main "how to work on this" in the `search` plugin, with basic details + a pointer to it in the other two new plugins.